src/main/java/com/hubspot/jinjava/util/ChunkResolver.java

```diff
@@ -182,14 +184,23 @@ public String resolveChunks() {
         }
         continue;
       }
-      prevChar = c;
+      setPrevChar(c);
       tokenBuilder.append(c);
     }
     miniChunkBuilder.append(resolveToken(tokenBuilder.toString()));
     chunks.add(resolveChunk(miniChunkBuilder.toString()));
     return chunks;
   }
 
+  private void setPrevChar(char c) {
+    if (c == '\\' && prevChar == '\\') {
+      // Backslashes cancel each other out for escaping when there's an even number.
+      prevChar = '\0';
```

Comment:
Here it's just set to the null character as `prevChar` is only used to check escaping. So setting it to the null character simply ignores the slash for escaping purposes, allowing any number of slashes to be back-to-back.